pkg/api/v1beta3/dynakube/telemetryservice_props.go

```diff
@@ -0,0 +1,39 @@
+package dynakube
+
+type TelemetryServiceProtocol string
+
+const (
+	TelemetryServiceOtlpProtocol   TelemetryServiceProtocol = "otlp"
```

Comment:
it's hard to read, should we name it <prtocol_name><type>: eg. `OtlpTelemetryServiceProtocol `